### PR TITLE
Fix for old Mac OS X

### PR DESCRIPTION
**Pull request by scottdraves**
close Issue #408 by copying the nib files to where pre-10.8 Mac OS X can find them, and also by including nginx that has been built on a 10.7 Mac.

### CODE REVIEW

**Comment by ildipo** [comment]
looks good. are we including the 'old' nginx also on newer macs?

**Comment by scottdraves** [comment]
yes fortunately they do have forward compatibility.

**Comment by scottdraves** [comment]
the version of nginx is the same (1.6.2) it's just linked with an old SDK.

File: core/build.gradle [hunk 1]
```diff
@@ -130,7 +130,8 @@ task (compileCssDebug, type: Exec) {
 
 task (copyNginx, type: Exec) {
   if (mac()) {
-    def brewLocation = 'which brew'.execute().text[0..-10]
+//  def brewLocation = 'which brew'.execute().text[0..-10] // use nginx from current machine
```
Comment (buob) [inline core/build.gradle]: We can probably kill this comment, huh?  Or are we going to go back to it in the near future?  Or does it add value to the new code?  If it's just to keep a record, we could always come back in source control if we need it, right?

**Pull request merged**